src/Components/EventTile/EventTile.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './EventTile.css'
+
+const EventTile = ({ events }) => {
+  return (
+    <div className='event-display'>
+      {events.map(event => (
+        <div className='event-tile' key={event.id}>
+          <div className="event-info">
+            <span className="event-name">{event.name}</span>
+            <span className="event-time">{event.time} o'clock</span>
+          </div>
+          </div>
+      ))}
+    </div>
+  );
+};
+
+export default EventTile;
```

Comment:
Good call creating a separate component for each event card!

Noting back to my previous comment - will need to add the date here, too, but could easily include that in the `event.time` span.